src/Mono.Android/Mono.Android.targets

```diff
@@ -177,7 +177,7 @@
       Inputs="$(TargetPath);@(ApiCompatibilityFiles)"
       Outputs="$(IntermediateOutputPath)CheckApiCompatibility.stamp">
     <CheckApiCompatibility
-        ApiCompatPath="$(XamarinAndroidSourcePath)\packages\microsoft.dotnet.apicompat\5.0.0-beta.20078.1\tools\net472\"
+        ApiCompatPath="$(NuGet_Packages)\microsoft.dotnet.apicompat\5.0.0-beta.20078.1\tools\net472\"
```

Comment:
This part feels like it'll fail if `$(NuGet_Packages)` isn't set, which is probably the case on Linux & macOS builds.

Reply:
I know that this is not full fix. I just want to point to location which I have to change for myself. I try different options, but not that good with Xamarin codebase to made changes. I technically can create separate MSBuild variable, but not sure that this is desired change. What to discuss first,